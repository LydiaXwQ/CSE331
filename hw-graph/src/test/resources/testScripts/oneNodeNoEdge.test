# Tests for a graph with one node and no edges.
# Test that a graph can be created, one node can be added, and the
# nodes and edges can be printed out.

# Create a graph
CreateGraph graph1

# Add a node
AddNode graph1 n1

# Print the nodes in the graph and the outgoing edges from n1
ListNodes graph1
ListChildren graph1 n1