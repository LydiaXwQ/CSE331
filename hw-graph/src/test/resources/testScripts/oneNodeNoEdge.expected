# Tests for a graph with one node and no edges.
# Test that a graph can be created, one node can be added, and the
# nodes and edges can be printed out.

# Create a graph
created graph graph1

# Add a node
added node n1 to graph1

# Print the nodes in the graph and the outgoing edges from n1
graph1 contains: n1
the children of n1 in graph1 are: